docs/src/test-ui-mode-js.md

```diff
@@ -108,7 +108,7 @@ Next to the Actions tab you will find the Metadata tab which will show you more
 
 ## Watch mode
 
-Next to the name of each test in the sidebar you will find an eye icon. Clicking on the icon will activate watch mode which will re-run the test when you make changes to it. You can watch a number of tests at the same time be clicking the eye icon next to each one or all tests by clicking the eye icon at the top of the sidebar. If you are using VS Code then you can easily open your test by clicking on the file icon next to the eye icon. This will open your test in VS Code right at the line of code that you clicked on.
```

Comment:
@Skn0tt, what do you think about this part?

The button has been moved to the Source tab, so I reflected the change [here](https://github.com/microsoft/playwright/pull/32907/files#diff-cd5a9688c278134c1b30e9e5db3afbbac143be7f95c531a14551ef04f7748673R63).

I can also open a separate MR for that if you believe it's not a bug.